collector/overhead_alert_metrics.go

```diff
@@ -128,9 +179,10 @@ func (r *ReconcileOverhead) Reconcile(ctx context.Context, request reconcile.Req
 				log.V(4).Info(fmt.Sprintf("filtering scheduling metric for %s with gap %v and total %v",
 					request.NamespacedName.String(), scheduleDuration, totalDuration))
 			}
-		} else {
```

Comment:
this is the path where we abort / skip this completed entry because it already has the throttled label ... no need to attempt ot throttle again